src/Add.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+function Add() {
+	return (
+			<h1>Add an item to your list here</h1>
+	)
+}
+
+export default Add;
```

Comment:
![image](https://user-images.githubusercontent.com/4306/69003514-561d8000-08b8-11ea-9063-6ddecbbcee14.png)

See the red icons at the end of the line? This is GitHub complaining that the file doesn't end with a newline (`\n`). **No need to fix this,** but just be aware that in some systems/languages, that lack of an extra return can cause unintended errors. It's a good habit to be in to always add one last return.

Reply:
Wait, did y'all fix this while I was adding the comment? I don't see the red icons anymore 🤔

Reply:
Oh. It was just GitHub not showing the icons in the other view. Sorry for the comment churn!